online/src/main/scala/ai/chronon/online/MetadataStore.scala

```diff
@@ -155,9 +155,17 @@ class MetadataStore(kvStore: KVStore, val dataset: String = ChrononMetadataKey,
     confPath.split("/").takeRight(3).mkString("/")
   }
 
-  // upload the materialized JSONs to KV store:
-  // key = <conf_type>/<team>/<conf_name> in bytes e.g joins/team/team.example_join.v1 value = materialized json string in bytes
-  def putConf(configPath: String): Future[Seq[Boolean]] = {
+  // derive team from path to file
+  def pathToTeam(confPath: String): String = {
+    //  // capture <conf_type>/<team> as key e.g joins/team
```

Comment:
Can you add an example of input and output of the function to the comments?

Reply:
@nikhilsimha updated